src/settings-web/src/components/GeneralSettings.tsx

```diff
@@ -116,19 +119,88 @@ export class GeneralSettings extends React.Component <any, any> {
           setting={{display_name: 'Start at login', value: this.state.settings.general.startup}}
           on_change={this.parent_on_change}
           ref={(input) => {this.startup_reference=input;}}
-          />
+        />
+        <ChoiceGroupSettingsControl
+          setting={{display_name: 'Chose Settigs color',
```

Comment:
```suggestion
          setting={{display_name: 'Chose Settings color',
```
Sorry spotted a typo in your gif